public/app/features/transformers/editors/HistogramTransformerEditor.tsx

```diff
@@ -1,11 +1,11 @@
-import React, { FormEvent, useCallback } from 'react';
+import React, { useCallback } from 'react';
 import { DataTransformerID, standardTransformers, TransformerRegistryItem, TransformerUIProps } from '@grafana/data';
-
 import {
   HistogramTransformerOptions,
   histogramFieldInfo,
 } from '@grafana/data/src/transformations/transformers/histogram';
-import { InlineField, InlineFieldRow, InlineSwitch, Input } from '@grafana/ui';
+import { InlineField, InlineFieldRow, InlineSwitch } from '@grafana/ui';
+import { NumberInput } from '../../dimensions/editors/NumberInput';
```

Comment:
bettter would be something like:
```suggestion
import { NumberInput } from '../grafana-ui/src/components/NumberInput/NumberInput.tsx';
```